collect_app/src/main/java/org/odk/collect/android/spatial/OsmMBTileSource.java

```diff
@@ -88,7 +88,6 @@ public static OsmMBTileSource createFromFile(File file) {
 
         int value;
```

Comment:
While you're here it might make sense to do the same thing for the other vars declared before they're initialized. We try to avoid that style. Doesn't need to be done across the codebase right now but since there's a cluster of them, it's easy to fix.